README.md

```diff
@@ -124,3 +124,33 @@ Further examples of all of the above can be found in `examples/`
 </table>
 
 For an extended description of our workflow, see https://gist.github.com/digitaljhelms/4287848
+
+### Enable Logging
```

Comment:
I wonder if there's a jupyter notebook we could stick this in. (I think it being in the README is fine though)